db/schema.rb

```diff
@@ -138,7 +138,7 @@
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
     t.integer  "belongs_to"
-    t.boolean  "is_common",  default: false
+    t.boolean  "is_common",  default: false, index: {name: "index_skills_on_is_common"}
```

Comment:
Updates to schema.rb were generated by rails/rake, so I'm going to leave it as-generated. 

Will double check on CI.